frontend/src/components/WorkProcessSection/index.jsx

```diff
@@ -14,43 +14,42 @@ const WorkProcess = () => {
     const { title, circleItem } = workProcessData;
 
     return (
-      <section
-        style={{ borderTopLeftRadius: '200px', marginTop: '5.625rem' }}
-        className="border-4 border-c1000 pt-24 pb-32 bg-c400  mt-20"
-      >
+      <section className="border-4 border-c1000 lg:pt-24 pt-16 pb-32 bg-c400 lg:rounded-tl-md lg:mt-24 mx-auto">
         <Heading
           as="h1"
           fontSize={HEADING_OPTIONS.FONT_SIZES.MEDIUM}
           color={HEADING_OPTIONS.COLOR.PRIMARY}
-          className="text-center mb-16"
+          className="text-center lg:mb-16 mb-16"
         >
           {title}
         </Heading>
         <div className="container relative">
-          <div className=" grid grid-cols-4 items-center justify-center">
+          <div className=" grid  lg:grid-cols-4 sm:grid-cols-1 items-center justify-center ">
```

Comment:
I think if we add the two columns style at the table view will be nicer

Reply:
this section must be sorted to show the steps in any project